imports/ui/QuestionDisplay.jsx

```diff
@@ -426,7 +426,7 @@ export class QuestionDisplay extends Component {
           : ''
         }
         
-        {(this.state.showCorrect || (q.sessionOptions && q.sessionOptions.correct)) && q.solution.length > 0 ? <div className='ql-question-solution'>Solution:<div className='ql-question-solution-content'>{WysiwygHelper.htmlDiv(q.solution)}</div></div> : ''}
+        {(this.state.showCorrect || (q.sessionOptions && q.sessionOptions.correct)) && q.solution.length > 0 && q.type !== QUESTION_TYPE.SA ? <div className='ql-question-solution'>Solution:<div className='ql-question-solution-content'>{WysiwygHelper.htmlDiv(q.solution)}</div></div> : ''}
```

Comment:
I think we still want to show the solution for short answer. What we need to remove is the extra box in the QuestionEditor that used to be used for the solution. Now that we have a solution field, we don't need the hack to hold the solution (which was to store the solution in the first option of what is normally used for the MC/MS questions).